packages/sdk-codegen/src/specConverter.ts

```diff
@@ -190,5 +246,313 @@ export const fixConversion = (
+
+/**
+ * Returns true spec is an OpenAPI specification
+ * @param spec to check
+ */
+export const isOpenApi = (spec: any) => spec.openapi !== undefined
+
+/**
+ * This is a post-fix operation for the OpenAPI converter which currently misses this type of conversion
+ *
+ * - Converts missing swagger collectionFormat values to OpenAPI styles
+ * - Flags OAS.requestBody as required or optional
+ *
+ * @param openApiSpec converted OpenAPI spec
+ * @param swaggerSpec original swagger spec that may contain missed conversions
+ * @returns modified openApiSpec and fix log in IConversionResults
+ */
+export const fixConversion = (
+  openApiSpec: string,
+  swaggerSpec: string
+): IConversionResults => {
+  return fixConversionObjects(JSON.parse(openApiSpec), JSON.parse(swaggerSpec))
+}
+
+/**
+ * Convert a swagger structure ref to an OpenAPI structure ref
+ *
+ * TODO update this to the typescript that understands replaceAll
+ * @param ref string reference to convert
+ */
+export const swapRef = (ref: string) =>
+  ref.replace(/#\/definitions\//g, '#/components/schemas/')
+
+/**
+ * Get the name of the structure from the reference string
+ * @param ref to partse
+ */
+export const structName = (ref: string) => {
+  if (!ref) return undefined
+  const parts = ref.split('/')
+  return parts[parts.length - 1]
+}
+
+/**
+ * Moves type, format, items to schema
+ * Converts collectionFormat to style
+ * @param param to convert
+ */
+export const convertParam = (param: any) => {
+  const schema: any = { type: param.type }
+  const result = { ...param }
+  delete result.type
+  if (param.format) {
+    schema.format = param.format
+    delete result.format
+  }
+  if (param.collectionFormat) {
+    result.style = openApiStyle(param.collectionFormat)
+    delete result.collectionFormat
+  }
+  if (param.items) {
+    schema.items = param.items
+    result.explode = false
+    delete result.items
+  }
+  result.schema = schema
+  return result
+}
+
+/**
+ * Convert responses for the right response formats
+ * @param responses response to convert
+ * @param formats mime formats
+ */
+export const convertResponses = (
+  responses: ArgValues,
+  formats = defaultMimeFormats
+) => {
+  Object.entries(responses).forEach(([code, response]) => {
+    responses[code] = {
+      description: response.description,
+    }
+    if (response.schema) {
+      const content = {}
+      formats.produces.forEach(
+        (format) => (content[format] = { schema: response.schema })
+      )
+      responses[code].content = content
+    }
+  })
+  return responses
+}
+
+/**
+ * Convert an operation
+ * Assign schemas in params, create request bodies, update $refs
+ * @param op operation to convert
+ * @param formats mime format consumer and producer
+ */
+export const convertOp = (op: ArgValues, formats = defaultMimeFormats) => {
+  // If keys need to be shuffled for file comparison reasons, uncomment this function
```

Comment:
my only concern about this pattern is that the commented code can get stale. If there really might be a use-case for key shuffling, maybe it's better to pass a flag into `convertOp` and write a test?

Reply:
I changed this code and verified the change has been pushed but GitHub is once again not detecting the change :confounded: